src/tests/conformance/query/get_by_hash.rs

```diff
@@ -74,3 +75,49 @@ async fn should_get_transaction_by_hash() {
     assert!(synth_node.expect_message(&check).await);
     node.stop().expect("unable to stop stateful node");
 }
+
+#[tokio::test]
+async fn should_query_for_transactions_after_have_transactions() {
+    // ZG-CONFORMANCE-009
+    // Create stateful node.
+    let target = TempDir::new().expect("unable to create TempDir");
+    let mut node = build_stateful_builder(target.path().to_path_buf())
+        .expect("unable to get stateful builder")
+        .start(false)
+        .await
+        .expect("unable to start stateful node");
+
+    // Wait for correct state and account data.
+    wait_for_state("proposing".into()).await;
```

Comment:
the possible states should probably be available via a dedicated `enum` instead, for added type safety; this is a nice-to-have and can be done later

Reply:
Added TODO